ambient.d.ts

```diff
@@ -22,9 +22,51 @@
  --------------
  ******/
 
+
 // for mojaloop there is lack for @types files
 // to stop typescript complains, we have to declare some modules here
 declare module '@mojaloop/central-services-logger'
+declare module '@mojaloop/central-services-metrics' {
+  import { Histogram } from 'prom-client'
+
+  interface metricOptionsType {
+    prefix: string
+    timeout: number
+  }
+  interface Metrics {
+
+    /**
+     * @function getHistogram
+     * @description Get the histogram values for given name
+     * @param {string} name - The name of the histogram to get. If the name doesn't exist, it creates a new histogram
+     * @param {string | undefined} help - Help description of the histogram (only used with creating a new histogram)
```

Comment:
does doc notation `string | undefined` means optional argument? so let use [] notation: https://jsdoc.app/tags-param.html
```suggestion
     * @param {string} [help] - Help description of the histogram (only used with creating a new histogram)
```

Reply:
and so on below

Reply:
Good point - I was borrowing from typescript definitions. I'll go ahead and update this.